dashboard/app/models/survey_result.rb

```diff
@@ -21,29 +21,32 @@ class SurveyResult < ApplicationRecord
   SYSTEM_DELETED = 'system_deleted'.freeze
 
   ETHNICITIES = {
-    "white" => "White",
-    "black" => "Black or African American",
-    "hispanic" => "Hispanic or Latino",
     "asian" => "Asian",
-    "native" => "Native Hawaiian or Other Pacific Islander",
-    "american_indian" => "American Indian or Alaska Native",
-    "other" => "Other/Unknown"
+    "black" => "Black / African American",
+    "hispanic" => "Hispanic / Latino / Latina",
+    "american_indian" => "Native American / Alaskan",
+    "hawaiian" => "Native Hawaiian / Pacific Islander",
+    "white" => "White",
+    "tr" => "Two or more races"
   }.freeze
 
   DIVERSITY_ATTRS = (ETHNICITIES.keys.map {|key| "diversity_#{key}"} + ['diversity_farm']).freeze
 
   NET_PROMOTER_SCORE_ATTRS = %w(nps_value nps_comment).freeze
 
+  # Note: diversity_native is deprecated. It corresponded to the "Native Hawaiian
```

Comment:
Nit: other is deprecated as well, yeah?

Reply:
Oh right, thanks for pointing that out.  I'm going to go ahead and merge but will update if I'm back in here again.